sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisErrorSuite.scala

```diff
@@ -149,7 +149,7 @@ class AnalysisErrorSuite extends AnalysisTest {
           UnresolvedAttribute("a") :: Nil,
           SortOrder(UnresolvedAttribute("b"), Ascending) :: Nil,
           UnspecifiedFrame)).as('window)),
-    "not supported within a window function" :: Nil)
+    "does not have any window functions" :: Nil)
```

Comment:
This is because an early analysis exception is thrown by rule ExtractWindowExpressions